lib/auth/apiserver.go

```diff
@@ -113,9 +113,6 @@ func NewAPIServer(config *APIConfig) (http.Handler, error) {
 	srv.GET("/:version/users/:user", srv.withAuth(srv.getUser))
 	srv.DELETE("/:version/users/:user", srv.withAuth(srv.deleteUser)) // DELETE IN: 5.2 REST method is replaced by grpc method with context.
 
-	// Generating keypairs
-	srv.POST("/:version/keypair", srv.withAuth(srv.generateKeyPair))
```

Comment:
We're certain nothing calls this, right?

Reply:
Well, `lib/client` did, but I removed that, I haven't been able to find anything else.